Documentation/cluster-crd.md

```diff
@@ -23,7 +23,8 @@ Settings can be specified at the global level to apply to the cluster as a whole
 - `versionTag`: The version (tag) of the `rook/rook` container that will be deployed. Upgrades are not yet supported if this setting is updated for an existing cluster, but upgrades will be coming.
 - `dataDirHostPath`: The host path where config and data should be stored for each of the services. If the directory does not exist, it will be created. Because this directory persists on the host, it will remain after pods are deleted.  Therefore, for test scenarios, the path must be deleted if you are going to delete a cluster and start a new cluster on the same hosts.  More details can be found in the Kubernetes [host path docs](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
 If this value is empty, each pod will get an ephemeral directory to store their config files that is tied to the lifetime of the pod running on that node. More details can be found in the Kubernetes [empty dir docs](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
-- `hostNetwork`: uses host network instead of using the SDN below the containers
+- `hostNetwork`: uses network of the hosts instead of using the SDN below the containers.
+- `monCount`: set the amount of mons to be started. The number must be uneven and between `1` and `9`. A good value for this is `3`.
```

Comment:
suggestion: use the word `odd` instead of `uneven`. And depending on the other discussion, the default is 3 again. :)